api-service/service/service.go

```diff
@@ -203,6 +210,8 @@ type APIService struct {
 	NewObjectID func() primitive.ObjectID
 
 	mockGetOracleDatabaseAgreements func(filters dto.GetOracleDatabaseAgreementsFilter) ([]dto.OracleDatabaseAgreementFE, error)
+	mockGetDatabaseLicensesCompliance func() ([]dto.LicenseCompliance, error)
```

Comment:
This is not how you must mock a method!
`mockGetOracleDatabaseAgreements` is an exception and should remain so!